### PR TITLE
Midnight Sun spoiler season day 3 cards!

### PR DESCRIPTION
**Pull request by plural**

### CODE REVIEW

File: v2/cards/steelskin_scarring.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  "cost": 1,
+  "deck_limit": 3,
+  "faction_id": "anarch",
+  "id": "steelskin_scarring",
+  "influence_cost": 2,
+  "is_unique": false,
+  "side_id": "runner",
+  "stripped_text": "Draw 3 cards. When this event is trashed from your grip or stack, you may draw 2 cards. ",
+  "stripped_title": "Steelskin Scarring",
+  "text": "Draw 3 cards.\nWhen this event is trashed from your grip or stack, you may draw 2 cards. ",
```
Comment (NoahTheDuke) [inline v2/cards/steelskin_scarring.json:12]: Ending space here and in stripped.

File: v2/printings/midnight_sun.json [hunk 2]
```diff
@@ -35,6 +35,18 @@
...
   },
+  {
+    "card_id": "steelskin_scarring",
+    "card_set_id": "midnight_sun",
+    "flavor": "Reactive implants reclaim the memories of our fallen comrades. Their sacrifice is our shield.",
+    "id": "33004",
+    "illustrator": "Elliott Birt",
+    "position": 4,
+    "printed_is_unique": false,
+    "printed_text": "Draw 3 cards.\nWhen this event is trashed from your grip or stack, you may draw 2 cards. ",
```
Comment (NoahTheDuke) [inline v2/printings/midnight_sun.json:46]: Also here

File: v2/printings/midnight_sun.json [hunk 3]
```diff
@@ -35,6 +35,18 @@
...
+    "quantity": 3,
+    "stripped_printed_text": "Draw 3 cards. When this event is trashed from your grip or stack, you may draw 2 cards. "
```
Comment (distributive) [inline v2/printings/midnight_sun.json:48]:
```suggestion
    "stripped_printed_text": "Draw 3 cards. When this event is trashed from your grip or stack, you may draw 2 cards."
```

File: v2/printings/midnight_sun.json (hunk 2 above)
Comment (distributive) [inline v2/printings/midnight_sun.json:46]:
```suggestion
    "printed_text": "Draw 3 cards.\nWhen this event is trashed from your grip or stack, you may draw 2 cards.",
```

File: v2/cards/steelskin_scarring.json (hunk 1 above)
Comment (distributive) [inline v2/cards/steelskin_scarring.json:12]:
```suggestion
  "text": "Draw 3 cards.\nWhen this event is trashed from your grip or stack, you may draw 2 cards.",
```

File: pack/ms.json [hunk 4]
```diff
@@ -60,6 +60,25 @@
...
+    "deck_limit": 3,
+    "faction_code": "anarch",
+    "faction_cost": 2,
+    "flavor": "Reactive implants reclaim the memories of our fallen comrades. Their sacrifice is our shield.",
+    "illustrator": "Elliott Birt",
+    "pack_code": "ms",
+    "position": 4,
+    "quantity": 3,
+    "side_code": "runner",
+    "stripped_text": "Draw 3 cards. When this event is trashed from your grip or stack, you may draw 2 cards. ",
```
Comment (distributive) [inline pack/ms.json:75]:
```suggestion
    "stripped_text": "Draw 3 cards. When this event is trashed from your grip or stack, you may draw 2 cards.",
```

File: pack/ms.json [hunk 5]
```diff
@@ -60,6 +60,25 @@
...
+    "stripped_title": "Steelskin Scarring",
+    "text": "Draw 3 cards.\nWhen this event is trashed from your grip or stack, you may draw 2 cards. ",
```
Comment (distributive) [inline pack/ms.json:77]:
```suggestion
    "text": "Draw 3 cards.\nWhen this event is trashed from your grip or stack, you may draw 2 cards.",
```

**Comment by plural** [comment]
Thanks!  I patched up the converter and tidied these issues up.

**Review by NoahTheDuke — APPROVED** [review]

**Review by distributive — APPROVED** [review]

**Pull request merged**